Jenkinsfile-e2e

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env groovy
+
+def dockerRegistry = "329802642264.dkr.ecr.eu-west-1.amazonaws.com"
+def nodeImageVersion = "10.16.0-1"
+def nodeImage = "${dockerRegistry}/bbc-news/node-10-lts:${nodeImageVersion}"
+
+def slackChannel = '#si_repo-simorgh'
+
+// Run latest every 3 hours, between 6 & 6, Monday - Friday
+def cron_string = env.BRANCH_NAME == "latest" ? "H 0 6-6/3 * * 1-5" : ""
```

Comment:
This should be all day every day. I would do it every 4 hours 24/7

Reply:
```suggestion
def cron_string = env.BRANCH_NAME == "latest" ? "H 0 */4 * * *" : ""
```